src/memkind.c

```diff
@@ -1042,6 +1043,25 @@ MEMKIND_EXPORT int memkind_create_pmem_with_config(struct memkind_config *cfg,
     return status;
 }
 
+MEMKIND_EXPORT int memkind_create_fixed(void *addr, size_t size,
+                                        memkind_t *kind)
+{
+    char name[32];
```

Comment:
Allow longer names. The longest name of static kinds is memkind_highest_bandwidth_local_preferred which is ca. 40 characters long. It's easy to imagine that one would like to use names with the similar char count.